bane-of-wargs.html

```diff
@@ -19,6 +19,9 @@
 	<p class="dialog">cd Bane-Of-Wargs/</p>
 	<p class="dialog">pip install -r requirements.txt</p>
 	<p class="dialog">python source/main.py</p>
+      <p></p>
+      <p>Here's the plot of the vanilla data:</p>
```

Comment:
```suggestion
      <p>The vanilla plot starts with this:</p>
```

Reply:
This is the resume of the intro storyline. The plot does not start with this dialog.

Reply:
Yes, but it also isn't the entire plot. Maybe:
```suggestion
      <p>The vanilla plot begins like this:</p>
```